server/models/friendRequestModel.js

```diff
@@ -0,0 +1,29 @@
+const { DataType: dt } = require("sequelize");
```

Comment:
Trevor's PR is going to be updated so make sure before you merge you have these models up to date with dev